tsl/test/expected/continuous_aggs_union_view-10.out

```diff
@@ -436,3 +436,121 @@ SELECT * FROM boundary_view ORDER BY time_bucket;
           40 | 400
 (4 rows)
 
+---- TEST union view with WHERE, GROUP BY and HAVING clause ----
+create table foo (a integer, b integer, c integer);
+select table_name FROM create_hypertable('foo', 'a', chunk_time_interval=> 10);
+NOTICE:  adding not-null constraint to column "a"
```

Comment:
May be put `not null` constraint on `a` in the table definition to avoid this unnecessary notice?